src/main/java/com/pmm/budgetcalculator/expensing/domain/expense/Expense.java

```diff
@@ -0,0 +1,28 @@
+package com.pmm.budgetcalculator.expensing.domain.expense;
+
+import com.pmm.budgetcalculator.foundation.BaseEntity;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+import javax.persistence.Entity;
+import java.time.LocalDateTime;
+
+
+@Entity
+@Data
+@EqualsAndHashCode(onlyExplicitlyIncluded = true, callSuper = true)
+@AllArgsConstructor
+
+public class Expense extends BaseEntity {
+    long expenseId;
+    String expenseName;
+    String expenseDescription;
+    LocalDateTime expenseTime;
+    double expenseAmount; //or BigDecimal
```

Comment:
BigDecimal when dealing with money is very good idea. double has less precision and can cut out some part (for example 0.000001) of your expense. It's not that bad? Multiply it by thousands expenses...